src/generators/minecraft-item/MinecraftItemGenerator.res

```diff
@@ -23,13 +23,44 @@ The generator supports four standard sizes:
 
 let images: array<Generator.imageDef> = [
   {id: "Background", url: Generator.requireImage("./images/Background.png")},
+  {id: "Title", url: Generator.requireImage("./images/Title.png")},
 ]
 
-let textures: array<Generator.textureDef> = TextureVersions.allTextureDefs
+let textures: array<Generator.textureDef> = Js.Array.concat(
+  TextureVersions.allTextureDefs,
+  [
+    {
+      id: "CenterFold",
+      url: Generator.requireImage("./item-textures/CenterFold.png"),
```

Comment:
Let's move this image to a `textures` directory, and we'll keep `item-textures` just for the item specific textures.

This is to minimise the likelihood of a particular technical problem occurring when we use textures.